web/public/mgmt/config.yml

```diff
@@ -3754,18 +3799,15 @@ collections:
               - { label: Section 5 - Third Header Text, name: section5ThirdHeaderText, widget: string }
               - { label: Section 5 - Third Supporting Text, name: section5ThirdSupportingText, widget: text }
               - { label: Section 6 - Header Text, name: section6Header, widget: string }
-              - { label: Card 1 - Header Line 1, name: card1HeaderLine1, widget: string }
-              - { label: Card 1 - Header Line 2, name: card1HeaderLine2, widget: string }
+              - { label: Card 1 - Header Line 1, name: card1Header, widget: string }
               - { label: Card 1 - Supporting Text, name: card1SupportingText, widget: string }
               - { label: Card 1 - Button Text, name: card1Button, widget: string }
               - { label: Card 1 - Button Link, name: card1ButtonLink, widget: string }
-              - { label: Card 2 - Header Line 2, name: card2HeaderLine1, widget: string }
-              - { label: Card 2 - Header Line 2, name: card2HeaderLine2, widget: string }
+              - { label: Card 2 - Header Line 2, name: card2Header, widget: string }
               - { label: Card 2 - Supporting Text, name: card2SupportingText, widget: text }
               - { label: Card 2 - Button Text, name: card2Button, widget: string }
               - { label: Card 2 - Button Link, name: card2ButtonLink, widget: string }
-              - { label: Card 3 - Header Line 3, name: card3HeaderLine1, widget: string }
-              - { label: Card 3 - Header Line 3, name: card3HeaderLine2, widget: string }
+              - { label: Card 3 - Header Line 3, name: card3Header, widget: string }
               - { label: Card 3 - Supporting Text, name: card3SupportingText, widget: text }
```

Comment:
[dust] - here is the only real changes in the file, all the rest is formatting which happened without my meddling